tools/fileconv/api3/strategy.go

```diff
@@ -6,23 +6,30 @@ import (
 	"github.com/amp-labs/connectors/common/handy"
 )
 
-// This strategy prunes URL Path that should be omitted during Schema extraction.
+// This strategy prunes URL Path that should be omitted or included during Schema extraction.
 // It allows hard coded endpoint Path, as well as simple star rules focusing on matching prefix or suffix.
+// First mode ignore endpoints and those matching suffixes, prefixes,
+// otherwise, only those that match the rules, will be used.
+//
 // Ex:
 // Basic:	/v1/orders	- ignores this path
 // Suffix:	*/batch		- ignores paths ending with batch
 // Prefix:	/v2/*		- ignores paths starting with v2.
 type ignorePathStrategy struct {
-	ignoreEndpoints handy.StringSet
-	prefixes        []string
-	suffixes        []string
+	endpoints handy.StringSet
+	prefixes  []string
+	suffixes  []string
+	// ignore flag determines the mode.
+	// It changes how we answer questions: Do we "ignore", or do we "include" given endpoint?
+	ignore bool
```

Comment:
This is hard to understand IMO. To make it clearer, can you split this into two strategies? One could be an ignoreObjects strategy, and one could be a `onlyObjects` strategy. They could be controlled via a flag respectively.